src/PrestaShopBundle/Command/ShopCreatorCommand.php

```diff
@@ -0,0 +1,247 @@
+            $customer = new Customer();
+            $customer->firstname = $faker->firstName;
+            $customer->lastname = $faker->lastName;
+            $customer->email = $faker->email;
+            $customer->passwd = '$2y$10$WzLnGz9j..JtTFcjfjoWr.8L/rw39NwovNRwPxf6yk/AYWcIj/1Au';
+            $customer->add();
+
+            $address = new Address();
+            $address->id_customer = $customer->id;
+            $address->alias = 'default';
+            $address->firstname = $customer->firstname;
+            $address->lastname = $customer->lastname;
+            $address->address1 = $faker->streetAddress;
+            $address->city = $faker->city;
+            $address->postcode = $faker->postcode;
+            $address->id_country = $faker->numberBetween(1, 10); // Modifier cette valeur selon les pays disponibles dans votre boutique
+            $address->phone = $faker->phoneNumber;
+            $address->dni = '1234567891012131';
+            $address->add();
+
+            $cart = new Cart();
+            $cart->id_customer = $customer->id;
+            $cart->id_address_delivery = $address->id;
+            $cart->id_address_invoice = $address->id;
+            $cart->id_currency = 1; // Modifier cette valeur selon les devises disponibles dans votre boutique
+            $cart->add();
+
+            for ($j = 0; $j < $faker->numberBetween(1, 5); ++$j) {
+                $randomProduct = $faker->randomElement($productIds);
+                $cart->updateQty($faker->numberBetween(1, 3), $randomProduct['id_product']);
+            }
+
+            $order = new Order();
+            $order->id_shop_group = $idShopGroup;
+            $order->id_shop = $idshop;
+            $order->id_cart = $cart->id;
+            $order->id_customer = $customer->id;
+            $order->id_address_delivery = $address->id;
+            $order->id_address_invoice = $address->id;
+            $order->id_currency = $cart->id_currency;
+            $order->id_carrier = 1;
+            $order->id_lang = $cart->id_lang;
+            $order->payment = 'Faker Payment';
+            $order->module = 'fakerpayment';
+            $order->total_paid = $cart->getOrderTotal(true, Cart::BOTH);
+            $order->total_paid_real = $cart->getOrderTotal(true, Cart::BOTH);
+            $order->total_paid_tax_incl = $cart->getOrderTotal(true, Cart::BOTH);
+            $order->total_paid_tax_excl = $cart->getOrderTotal(false, Cart::BOTH);
+            $order->total_products = $cart->getOrderTotal(false, Cart::ONLY_PRODUCTS);
+            $order->total_products_wt = $cart->getOrderTotal(true, Cart::ONLY_PRODUCTS);
+            $order->total_shipping = 0;
+            $order->secure_key = md5('test');
+            $order->carrier_tax_rate = 0;
+            $order->conversion_rate = 1;
+
+            $order->add();
+
+            $orderStatus = new OrderState(3); // 3 est l'ID du statut de la commande "en cours de préparation"
```

Comment:
We should generate more randomized data so that in BO we could filter orders by state, etc.